ios/GstPlay/gst_ios_init.h

```diff
@@ -15,6 +15,7 @@ G_PASTE(g_io_module_, G_PASTE(name, _load_static)) ()
  * You can also enable individual plugins. See gst_ios_init.c to see their names
  */
 
+#define GST_IOS_PLUGINS_GES
```

Comment:
This one can be commented out too. That's basically the new GST_IOS_PLUGINS_EDITING